File: c#/OOP/ExRectangular.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OOP
{
    class ExRectangular
    {
        private int height;
        private int width;
        public ExRectangular()
        {
            width = 1;
            height = 1;
        }
        public ExRectangular(int height, int width)
        {
            this.height = height;
            this.width = width;
        }
        public int Height
        {
            set
            {
                this.height = value;
            }
            get
            {
                return this.height;
            }
        }
        public int Width
        {
            set
            {
                this.width = value;
            }
            get
            {
                return this.width;
            }
        }
        public override string ToString()
        {
            return "Width: " + this.width + " Height: " + this.height;
        }

    }
}

File: Java/OOPs/src/OOPs/ExRectangular.java
package OOPs;

public class ExRectangular {
	
	private int height;
	private int width;
	public int getHeight() {
		return height;
	}
	public void setHeight(int height) {
		this.height = height;
	}
	public int getWidth() {
		return width;
	}
	public void setWidth(int width) {
		this.width = width;
	}
	public ExRectangular() {
		System.out.println("Contructor khong co doi so!@");
	}
	public ExRectangular(int height, int width) {
		this.height = height;
		this.width = width;
	}
	public int  acreage() {
		return this.height*this.width;
	}
	public int perimeter() {
		return (this.height+this.width)*2;
	}
	@Override
	public String toString() {
		return "ExRectangular [height=" + height + ", width=" + width + "]";
	}

}
